File: 453. Minimum Moves to Equal Array Elements.py
class Solution(object):
    def minMoves(self, nums):
        
        """
        ref:http://blog.csdn.net/yeqiuzs/article/details/53078666
        :type nums: List[int]
        :rtype: int
        """
        return sum(nums) - min(nums) * len(nums)

File: 453. Minimum Moves to Equal Array Elements/453. Minimum Moves to Equal Array Elements.c
int minMoves(int* nums, int numsSize) {
    int i = 0, sum = 0, min = nums[0];
    while(i < numsSize){
        sum += nums[i];
        min = nums[i] < min ? nums[i] : min;
        
    }
    return sum - min * numsSize;
}
